_data/authors.yml

```diff
@@ -24,14 +24,21 @@ brandon_chan:
 ciaran_schutte:
   name: Ciarán Schütte
   position: Software Developer
-  info:
+  info: Ciarán, a full stack developer, tried to get an AI to write his bio. He just couldn’t get it to write anything sensible, so instead he chose to write the bio himself using JavaScript and javascript. Especially the bit about delicious garlic.
   avatar: ciaran_schutte.svg
   github: ciaranschutte
 
+dan_demaria:
+  name: Dan DeMaria
+  position: Software Developer III
```

Comment:
suggest removing "III":
- it is inconsistent throughout (some have nothing, some have III, some have 3
- it doesn't mean anything outside of OICR

Reply:
Got it!

Reply:
In general I would not put levels, just role descriptions. This isn't a registry of our HR job titles.